src/crimson/os/seastore/lba_manager/btree/lba_btree.h

```diff
@@ -212,16 +218,20 @@ class LBABtree {
    */
   iterator_fut upper_bound_right(
     op_context_t c,
-    laddr_t addr) const
+    laddr_t addr,
+    bool need_count) const
   {
     return lower_bound(
       c, addr
-    ).si_then([c, addr](auto iter) {
+    ).si_then([c, addr, need_count](auto iter) {
       if (iter.is_begin()) {
 	return iterator_fut(
 	  interruptible::ready_future_marker{},
 	  iter);
       } else {
+	if (need_count) {
+	  ++LBABtree::lba_tree_inner_stats.num_iter_prevs;
```

Comment:
I think this metric doesn't provide very useful information for the goal, as we can directly see from the code that, if no transaction conflict happens, each `allocate_extent()` will increment this metric only once.